includes/batch.inc

```diff
@@ -126,7 +128,19 @@ function _drush_backend_batch_process($command = 'batch-process', $args = [], $o
 
     // Assign an arbitrary id: don't rely on a serial column in the 'batch'
     // table, since non-progressive batches skip database storage completely.
-    $batch['id'] = \Drupal::database()->nextId();
+    /** @var \Drupal\Core\Batch\BatchStorageInterface $batch_storage */
+    $batch_storage = \Drupal::service(BatchStorageInterface::class);
```

Comment:
Aliases been added in 10.1 so this will fail on earlier core, please use `batch.storage` as service name

Reply:
dgo.to/3049525